elementary/config/config.py

```diff
@@ -68,3 +75,12 @@ def has_aws(self) -> bool:
     @property
     def has_gcs(self):
         return self.gcs_bucket_name and self.google_service_account_path
+
+    def validate_elementary_profile(self):
+        profiles_path = os.path.join(self.profiles_dir, 'profiles.yml')
+        try:
+            profiles_yml = OrderedYaml().load(profiles_path)
```

Comment:
I'd check explicitly if the path exists and if not return False, I'd return False if the elementary profile was not found

Reply:
It's a more Pythonic approach.
_It's easier to ask for forgiveness than permission._

Reply:
If you returned a `boolean`, how would you know if it's not in `track-cli-exception`? 😉